### DISCUSSION TITLE
How to relay message to all connected clients except the one send the message ?

### DISCUSSION THREAD

**Discussion by Waterloo** [khoih-prog/WebSockets_Generic#25] [Q&A]
I looked into brodcastTXT built-in function but it broadcasts to all connected clients.

**Comment by khoih-prog** [comment]
The `broadcastTXT()` funcrion is designed for the `WebSockets Server` to `send text data **to all connected clients**`, not to the Server itself.

If it doesn't fit your purpose, it's still easy to write a special function for your **special use-case**, if you choose to go that hard way.
